templates/expo-template-bare-minimum/android/build.gradle

```diff
@@ -1,28 +1,17 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext {
-        buildToolsVersion = findProperty('android.buildToolsVersion') ?: '35.0.0'
-        minSdkVersion = Integer.parseInt(findProperty('android.minSdkVersion') ?: '24')
-        compileSdkVersion = Integer.parseInt(findProperty('android.compileSdkVersion') ?: '35')
-        targetSdkVersion = Integer.parseInt(findProperty('android.targetSdkVersion') ?: '35')
```

Comment:
since we now officially remove these in bare template. does it break expo-build-properties?

Reply:
you are so right. shame on me for the expo-build-properties author to forget this 🙈